addons/radar/XEH_preInit.sqf

```diff
@@ -80,6 +80,20 @@ if (_acre || _tfar) then {
     GVAR(showSpeaking_replaceIcon) = false;
 };
 
+if (isClass (configfile >> "CfgPatches" >> "ace_medical_engine")) then {
```

Comment:
Switch out "ace_medical_engine" to "ace_medical".

Then it supports old ace medical and new ace medical. There are still people playing with the old medical.